specifications/13-constraint-operators.json

```diff
@@ -0,0 +1,301 @@
+                        "parameters": {},
+                        "constraints": [
+                            {
+                                "contextName": "email",
+                                "operator": "STR_ENDS_WITH",
+                                "values": ["@some-email.com"]
+                            }
+                        ]
+                    }
+                ]
+            },
+            {
+                "name": "F3.endsWith.ignoringCase",
+                "description": "endsWith",
+                "enabled": true,
+                "strategies": [
+                    {
+                        "name": "default",
+                        "parameters": {},
+                        "constraints": [
+                            {
+                                "contextName": "email",
+                                "operator": "STR_ENDS_WITH",
+                                "values": ["@some-email.com"],
+                                "caseInsensitive": true
+                            }
+                        ]
+                    }
+                ]
+            },
+            {
+                "name": "F4.contains",
+                "description": "contains",
+                "enabled": true,
+                "strategies": [
+                    {
+                        "name": "default",
+                        "parameters": {},
+                        "constraints": [
+                            {
+                                "contextName": "email",
+                                "operator": "STR_CONTAINS",
+                                "values": ["email"]
+
+                            }
+                        ]
+                    }
+                ]
+            },
+            {
+                "name": "F4.contains.inverted",
+                "description": "contains",
+                "enabled": true,
+                "strategies": [
+                    {
+                        "name": "default",
+                        "parameters": {},
+                        "constraints": [
+                            {
+                                "contextName": "email",
+                                "operator": "STR_CONTAINS",
+                                "values": ["email"],
+                                "inverted": true
+
+                            }
+                        ]
+                    }
+                ]
+            },
+            {
+                "name": "F5.numEq",
+                "description": "contains",
+                "enabled": true,
+                "strategies": [
+                    {
+                        "name": "default",
+                        "parameters": {},
+                        "constraints": [
+                            {
+                                "contextName": "someValue",
+                                "operator": "NUM_EQ",
+                                "value": 12
```

Comment:
We might consider to use string values only, and have the SDK parse it to a number type.

Reply:
I think we're going to have to if we don't want to break the api contract with earlier sdks, but that from the operator, the sdk's will have to do the parsing.

Reply:
lets keep it as a string.